utils/machine/machine.go

```diff
@@ -34,12 +34,18 @@ import (
 
 // The utils/machine package contains functions that extract machine-level specs.
 
-var cpuRegExp = regexp.MustCompile("processor\\t*: +([0-9]+)")
-var coreRegExp = regexp.MustCompile("core id\\t*: +([0-9]+)")
-var nodeRegExp = regexp.MustCompile("physical id\\t*: +([0-9]+)")
-var CpuClockSpeedMHz = regexp.MustCompile("cpu MHz\\t*: +([0-9]+.[0-9]+)")
-var memoryCapacityRegexp = regexp.MustCompile("MemTotal: *([0-9]+) kB")
-var swapCapacityRegexp = regexp.MustCompile("SwapTotal: *([0-9]+) kB")
+var (
+	cpuRegExp            = regexp.MustCompile("processor\\t*: +([0-9]+)")
+	coreRegExp           = regexp.MustCompile("core id\\t*: +([0-9]+)")
+	nodeRegExp           = regexp.MustCompile("physical id\\t*: +([0-9]+)")
+	CpuClockSpeedMHz     = regexp.MustCompile("cpu MHz\\t*: +([0-9]+.[0-9]+)")
+	memoryCapacityRegexp = regexp.MustCompile("MemTotal: *([0-9]+) kB")
+	swapCapacityRegexp   = regexp.MustCompile("SwapTotal: *([0-9]+) kB")
+
+	cpuClockSpeedMHz = regexp.MustCompile("clock\\t*: +([0-9]+.[0-9]+)MHz")
```

Comment:
Unused? Did you mean to remove the lines where this was used below? Also, consider differentiating the variable name more from `CpuClockSpeedMHz`